network/src/peer_manager/mod.rs

```diff
@@ -40,46 +49,45 @@ mod tests;
 
 pub use self::error::PeerManagerError;
 
-/// Notifications about new/lost peers.
-#[derive(Debug)]
-pub enum PeerManagerNotification<TSubstream> {
-    NewPeer(PeerId, Multiaddr),
-    LostPeer(PeerId, Multiaddr),
-    NewInboundSubstream(PeerId, NegotiatedSubstream<TSubstream>),
-}
-
 /// Request received by PeerManager from upstream actors.
 #[derive(Debug)]
-pub enum PeerManagerRequest<TSubstream> {
+pub enum PeerManagerRequest {
```

Comment:
Maybe call this Peer or NetworkPeer?

Reply:
With this diff, it looks like this enum and `PeerManagerNotification` are basically the same enum as the previous `NetworkRequest`/`NetworkNotification` enums. Maybe we should use those names again?

Or do you mean like since this def lives inside the `peer_manager` namespace, why do we name it `PeerManagerRequest` instead of just `Request`, and other modules use `peer_manager::Request`? I think some things, like rpc used to do that.

Reply:
There was some issue with placing this comment appropriately and I should have clarified that. Your conclusions are reasonable though. 1) the naming of the thing should more accurately match what it is encapsulating. and 2) avoid name duplication if possible (i.e., peer_manager::Request).